app/src/services/namex-services.ts

```diff
@@ -28,18 +27,16 @@ axiosNamex.interceptors.request.use(
     config.headers.common['BCREG-User-Phone'] = sessionStorage.getItem('BCREG-phoneNumber')
     config.headers.common['BCREG-User-Email'] = sessionStorage.getItem('BCREG-emailAddress')
     config.headers.common['App-Name'] = pkg.name
-    config.headers.common['X-Apikey'] = process.env.VUE_APP_NAMEX_API_KEY || ''
-    // eslint-disable-next-line no-console
-    // console.log('in interceptor, common headers: ', config?.headers?.common)
```

Comment:
I deleted some of these that were already commented out.

Below, I commented out misc other console logs.